validator/val_utils.c

```diff
@@ -393,6 +393,16 @@ enum sec_status
 val_verify_rrset_entry(struct module_env* env, struct val_env* ve,
         struct ub_packed_rrset_key* rrset, struct key_entry_key* kkey,
 	char** reason, sldns_pkt_section section, struct module_qstate* qstate)
+{
+	return val_verify_rrset_entry_ede(
+			env, ve, rrset, kkey, reason, NULL, section, qstate);
+}
+
+enum sec_status 
+val_verify_rrset_entry_ede(struct module_env* env, struct val_env* ve,
+        struct ub_packed_rrset_key* rrset, struct key_entry_key* kkey,
+	char** reason, sldns_ede_code *reason_bogus,
+	sldns_pkt_section section, struct module_qstate* qstate)
```

Comment:
I would say that this function is not needed. We can just use `val_verify_rrset_entry` and pass NULL in the places where *reason_bogus is not relevant (there are only a few).